compiler/rustc_middle/src/mir/syntax.rs

```diff
@@ -1110,15 +1110,39 @@ pub enum NullOp {
     AlignOf,
 }
 
-#[derive(Copy, Clone, Debug, PartialEq, Eq, TyEncodable, TyDecodable, Hash, HashStable)]
+#[derive(
+    Copy,
+    Clone,
+    Debug,
+    PartialEq,
+    Eq,
+    PartialOrd,
+    Ord,
+    TyEncodable,
+    TyDecodable,
+    Hash,
+    HashStable
+)]
 pub enum UnOp {
     /// The `!` operator for logical inversion
     Not,
     /// The `-` operator for negation
     Neg,
 }
 
-#[derive(Copy, Clone, Debug, PartialEq, PartialOrd, Eq, TyEncodable, TyDecodable, Hash, HashStable)]
+#[derive(
+    Copy,
+    Clone,
+    Debug,
+    PartialEq,
+    PartialOrd,
+    Ord,
+    Eq,
+    TyEncodable,
+    TyDecodable,
+    Hash,
+    HashStable
+)]
```

Comment:
can you split that derive into two, to get a better formatting

(same for all other `derive`s in your pr)